examples/currency-quotes-scrapper/.meteor/packages

```diff
@@ -16,24 +16,5 @@ standard-minifiers      # JS/CSS minifiers run for production mode
 es5-shim                # ECMAScript 5 compatibility for older browsers.
 ecmascript              # Enable ECMAScript2015+ syntax in app code
 
-
-accounts-github
-accounts-ui
-aldeed:autoform
-aldeed:collection2
-aldeed:simple-schema
-check
-http
-kadira:blaze-layout
-kadira:flow-router
-less
-momentjs:moment
-mquandalle:jade
-npm-packages
-twbs:bootstrap
-random
-reactive-var
-okgrow:analytics
-meteorhacks:kadira
-meteorhacks:zones
-reactive-dict
+insecure                # Allow all DB writes from clients (for prototyping)
```

Comment:
@kdanninger nope I did not mean to, but it also does not matter since we do not have any DB for the examples.... just minimongo.